apps/studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -320,9 +320,16 @@ export const genChartQuery = (
   filters: Filters
 ) => {
   const [startOffset, trunc] = calcChartStart(params)
-  const where = genWhereStatement(table, filters)
+  let where = genWhereStatement(table, filters)
   const errorCondition = getErrorCondition(table)
 
+  // pg_cron logs are a subset of postgres logs
+  // to calculate the chart, we need to query postgres logs
+  if (table === LogsTableName.PG_CRON) {
+    table = 'postgres_logs'
+    where = `where (parsed.application_name = 'pg_cron' OR event_message LIKE '%cron job%')`
```

Comment:
hmm what are the chances of non-pg_cron events ending up in the stream unexpectedly?